src/raw_logging.cc

```diff
@@ -59,7 +59,7 @@
 # include <unistd.h>
 #endif
 
-#if defined(HAVE_SYSCALL_H) || defined(HAVE_SYS_SYSCALL_H)
+#if !defined(ANDROID) && (defined(HAVE_SYSCALL_H) || defined(HAVE_SYS_SYSCALL_H))
```

Comment:
I think you can use __NR_write. I think Binoic doesn't consider __NR_* are public APIs and the code may not compile in future, but I think we really don't want raw_logging to modify errno.